### DISCUSSION TITLE
Auto Conversion from Markdown: `title` Value of `img` HTML Tag as `figcaption` for Parent Tag (`figure`)

### DISCUSSION THREAD

**Discussion by nurbaitifitriyani** [mecha-cms/mecha#208] [Q&A]
Sebuah konten markdown berisi:

```md
![Xu Bing](/lot/asset/jpg/museum-macan-alat-pembuatan-karya-xu-bing-untuk-pengunjung.jpg "Alat Pembuatan Karya Xu Bing Untuk Pengunjung")

**Alat Pembuatan Karya Xu Bing Untuk Pengunjung**
```
Akan dirender sebagai HTML:

```html
<figure class="figure">
	<img alt="Alat Pembuatan Karya Xu Bing Untuk Pengunjung" loading="lazy" src="/lot/asset/jpg/museum-macan-alat-pembuatan-karya-xu-bing-untuk-pengunjung.jpg" title="Alat Pembuatan Karya Xu Bing Untuk Pengunjung">
</figure>

<p><strong>Alat Pembuatan Karya Xu Bing Untuk Pengunjung</strong></p>
```
Rasanya kurang lengkap kalau ada tag `figure` tanpa disertai `figcaption` dan mubadzir sekali kalau ada tag `p` tebal (agak kurang semantik. Juga membuang-buang tenaga untuk merepetisi kalimat dalam sebuah paragraf tebal. 

Sementara `alt` dalam gambar tetap unik, saya ingin mengambil nilai `title` dari tag `img` untuk dijadikan sebagai `figcaption`. Apakah ini bisa dilakukan via `hook` atau menyunting salah satu ekstensi? Bagaimanakah caranya?

**Comment by taufik-nurrohman** [comment]
Coba baca `README.md` di [repositori ini](https://github.com/taufik-nurrohman/parsedown-extra-plugin) mba.

**Reply by nurbaitifitriyani** [reply]
Baik, Mas. Sudah teratasi. Saya tidak detail membuka-buka ekstensi tersebut sebelumnya dan melewatkan `README.md`.

Untuk *featured image* di panel, apakah bisa dikasih *figcaption*? Saya mengakalinya dengan membuat `meta.data` (seperti diskusi [ini](https://github.com/mecha-cms/mecha/discussions/203#discussioncomment-2516533)) dan menyelipkan `figcaption` untuk saya terapkan di `featured image`.

**Reply by taufik-nurrohman** [reply]
> Untuk _featured image_ di panel, apakah bisa dikasih _figcaption_?

Sementara masih bukan menjadi prioritas mba.
